### DISCUSSION TITLE
how to pause all cron jobs

### DISCUSSION THREAD

**Discussion by tacman** [dokku/dokku#7143] [Q&A]
I need to pause all cron jobs while doing some maintenance, but don't want to rebuild the image with a modified app.json.  Is there a way to simple pause the jobs and then restart them, without rebuilding?

I just mean pausing the schedule, the actual jobs that are running will need to be stopped separately, but I don't want them restarting until I'm ready.

Thanks.

**Comment by josegonzalez** [comment]
There isn't a way to do that at this time. It seems reasonable though to disable cron for a given app. You could file a feature request outlining how this might work and we can iterate on a proposed solution there before implementing.

**Reply by tacman** [reply]
Thanks, I've proposed it in #7145
